adapters/nobid/nobid.go

```diff
@@ -0,0 +1,133 @@
+package nobid
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// NoBidAdapter - NoBid Adapter definition
+type NoBidAdapter struct {
+	endpoint string
+}
+
+// NewNoBidBidder Initializes the Bidder
+func NewNoBidBidder(endpoint string) *NoBidAdapter {
+	return &NoBidAdapter{
+		endpoint: endpoint,
+	}
+}
+
+type nobidParams struct {
+	SiteID string `json:"SiteID"`
+}
+
+// MakeRequests Makes the OpenRTB request payload
+func (a *NoBidAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+
+	var errors []error
```

Comment:
You actually don't need the `errors` variable, since you return errors immediately on failure.

Reply:
Done

Reply:
Pull request for the documentation: https://github.com/prebid/prebid.github.io/pull/2452